JASP-Engine/JASP/R/regressionlogistic.R

```diff
@@ -378,210 +675,500 @@ RegressionLogistic <- function(dataset=NULL, options, perform="run",
+    glmObj <- .reglogisticComputeModel(jaspResults, dataset, options)
+    mObj   <- glmObj[[length(glmObj)]]
+    m <- .confusionMatrix(mObj, cutoff = 0.5)[["metrics"]]
+    rows <- list(
+      list(met = "AUC",         val = m[["AUC"]]),
+      list(met = "Sensitivity", val = m[["Sens"]]),
+      list(met = "Specificity", val = m[["Spec"]]),
+      list(met = "Precision",   val = m[["Precision"]]),
+      list(met = "F-measure",   val = m[["F"]]),
+      list(met = "Brier score", val = m[["Brier"]]),
+      list(met = "H-measure",   val = m[["H"]])
+    )
+  } else
     rows <- list(
-      list(model = ".", param = ".", est = ".", bias = ".", se = ".", cilo = ".", ciup = ".")
+      list(met = "AUC",         val = "."),
+      list(met = "Sensitivity", val = "."),
+      list(met = "Specificity", val = "."),
+      list(met = "Precision",   val = "."),
+      list(met = "F-measure",   val = "."),
+      list(met = "Brier score", val = "."),
+      list(met = "H-measure",   val = ".")
     )
+  
+  # determine which scores we need
+  scrNeed  <- with(options, c(AUC, Sens, Spec, Prec, Fmsr, BrierScr, Hmsr))
+  rows     <- rows[scrNeed]
+  container[["performanceMetrics"]]$addRows(rows)
+}
+
+# Plots
+.reglogisticResidualPlotContainer <- function(jaspResults, options){
+  if(!options$predictedPlotOpt && !options$predictorPlotsOpt && 
+     !options$squaredPearsonPlotOpt)
+    return()
+  if (is.null(jaspResults[["residualPlots"]])) {
+    container <- createJaspContainer("Residual plots")
+    container$dependOn(optionsFromObject = jaspResults[["modelSummary"]],
+                       options           = c("residualType"))
+    jaspResults[["residualPlots"]] <- container
   }
+}
+
+.reglogisticEstimatesPlot <- function(jaspResults, dataset, options, ready){
+  if(!options$estimatesPlotsOpt || !ready)
+    return()
+  container <- createJaspContainer("Estimates plots")
+  jaspResults[["estimatesPlot"]] <- container
+  # Compute/Get Model
+  glmObj <- .reglogisticComputeModel(jaspResults, dataset, options)
+  mObj   <- glmObj[[length(glmObj)]]
+  predictors <- character(0)
+  for (term in options$modelTerms)
+    if (length(term$components) == 1 && 
+        (is.null(term$isNuisance) || !term$isNuisance))
+      predictors <- c(predictors, term$components)
   
-  footnotes <- .newFootnotes()
-  if(ci.fails){
-    .addFootnote(footnotes,
-                 symbol = "<i>Note.</i>", 
-                 text = "Some confidence intervals could not be computed. Possibly too few bootstrap replicates.")
+  if (length(predictors) > 0 && !is.null(glmObj)) {
+    # plot only predictors selected in the final model
+    predictors <- predictors[.v(predictors) %in% attr(mObj[["terms"]], 
+                                                      "term.labels")]
+    for(pred in predictors) {
+      estimatesPlot <- createJaspPlot(title = pred, width = 480, height = 320)
+      estimatesPlot$dependOn(optionsFromObject = jaspResults[["modelSummary"]],
```

Comment:
Nitpick: this is not different between plots, so why not put it on the container instead